notebooks/COS/LSF/LSF.ipynb

```diff
@@ -0,0 +1,1525 @@
+    "    except: # For other params\n",
+    "        value = fuvHeader0[hdrKeyword] # Save the key/value pairs to the dictionary\n",
+    "        param_dict[hdrKeyword] = value\n",
+    "    print(f\"{hdrKeyword} = {value}\") # Print the key/value pairs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Now that you know *which* LSF files you need, we'll go gather them from the COS team's website:**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font size=\"4 \">**Where can you find the LSF files?**</font>\n",
+    "The COS team maintains up-to-date LSF files on the [COS Spectral Resolution page](https://www.stsci.edu/hst/instrumentation/cos/performance/spectral-resolution). Opening up this link leads to a page like that shown in Fig. 1.1, where the LSF files are discussed in detail. The bottom part of this page has links to all the relavent files. The links at the top of the page will take you to the relevant section. In Fig. 1.1, we have circled in black the link to the section pertaining to our data: FUV at the Lifetime Position: 3.\n",
+    "\n",
+    "### <center> Fig 1.1: Screenshot of the COS Spectral Resolution Site</center>\n",
+    "<center><img src=./figures/LSFHomepage.png width =\"900\" title=\"COS Spectral Resolution Site\"> </center>\n",
+    "\n",
+    "Clicking on the circled link takes us to the table of hyperlinks to all the files perataining to data taken with the FUV, Lifetime Postition 3 configutation, shown in Fig. 1.2:\n",
+    "\n",
+    "### <center>Fig 1.2: Screenshot of the COS Spectral Resolution Site - Focus on LP-POS 3</center>\n",
+    "<center><img src=./figures/LSFHomepage2.png width =\"900\" title=\"COS Spectral Resolution Site - Lifetime Position 3\"> </center>\n",
+    "\n",
+    "Circled in solid red is the button to download the LSF file we need for our data with CENWAVE = 1291. Circled in dashed black is the corresponding CDSF.\n",
+    "\n",
+    "You can click on the solid red circled button to download the LSF file and move it to this current working directory **or** right-click and select \"Download Linked File As...\" and download directly to this directory.\n",
+    "* *This step may look somewhat different depending on your browser*\n",
+    "* *If you're unsure of the current working directory, un-comment the `!pwd` statement in the next cell, and run the cell.*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !pwd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These files can also be downloaded programattically from with python, with the function we define below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def fetch_files(det, grating, lpPos, cenwave, disptab):\n",
```

Comment:
No change needed now but an idea for the future: maybe this or similar should be in `costools`?  Seems generically useful...